buildSrc/src/main/java/Dependencies.kt

```diff
@@ -32,6 +34,7 @@ object Versions {
 object Dependencies {
     const val gradle_build_tool = "com.android.tools.build:gradle:${Versions.gradleBuildTool}"
     const val databinding = "androidx.databinding:compiler:${Versions.dataBinding}"
+    const val  appCompat= "androidx.appcompat:appcompat:${Versions.appCompat}"
```

Comment:
```suggestion
    const val appCompat = "androidx.appcompat:appcompat:${Versions.appCompat}"
```

just a quick fix in terms of space to maintain consistency with other adjacent lines of code ;)
So glad you understood what I meant earlier 🎉